__tests__/src/services/ClientService.test.ts

```diff
@@ -7,28 +7,26 @@
 
 import { ServiceCategory, clientService } from '../../../src/services/ClientService';
 import supabaseClient from '../../../utils/supabaseClient';
-describe('ClientService', () => {
 
+describe('ClientService', () => {
   it('should get service categories', async () => {
-    const response = { data: [new ServiceCategory({})], error: null }
+    const response = { data: [new ServiceCategory({})], error: null };
     const mockQueryBuilder = {
       select: jest.fn(() => Promise.resolve(response)),
     };
-    const fromSpy = jest.spyOn(supabaseClient, "from")
-      .mockReturnValue(mockQueryBuilder as any)
-    const selectSpy = jest.spyOn(mockQueryBuilder, "select")
-      .mockReturnValue(response as any)
+    const fromSpy = jest.spyOn(supabaseClient, 'from')
```

Comment:
In general its better to completely mock the calls to functions outside of the scope of your test. `spyOn` will still allow the underlying supabaseClient.from code to run, whereas mocking will completely isolate the test code from the actual implementation and reduce the chances of side-effects from simple loading the external module.

Reply:
I think for this PR I'll leave it because I was just making linting changes, but I'll make a note and we can have a PR in the future which corrects this